lib/windows/handle.rb

```diff
@@ -9,7 +9,7 @@ module Handle
 
     private
 
-    INVALID_HANDLE_VALUE           = 0xFFFFFFFF
+    INVALID_HANDLE_VALUE           = (1<<['a'].pack('p').length*8)-1
```

Comment:
I confirmed it works.

C:\Ruby22-x64\bin>irb
irb(main):001:0>  (1<<['a'].pack('p').length*8)-1
=> 18446744073709551615

C:\Ruby22\bin>irb
irb(main):001:0>  (1<<['a'].pack('p').length*8)-1
=> 4294967295